File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        map<int,int>map;
        for(int x : nums) {
            map[x]++;
        }
        for(int x : nums) {
            if(map[x] > 1) return true;
        }
        return false;
    }
};

File: Data Structures & Algorithms/squares-of-a-sorted-array/submission-0.py
class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        
        res = []
        for i in nums:
            res.append(i*i)
        for j in range(len(nums)):
            nums[j] = res[j]

        nums.sort()
        return nums